vendor/README.md

```diff
@@ -420,6 +421,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Python:** [tree-sitter/tree-sitter-python](https://github.com/tree-sitter/tree-sitter-python) 🐌
 - **Python console:** [MagicStack/MagicPython](https://github.com/MagicStack/MagicPython)
 - **Python traceback:** [MagicStack/MagicPython](https://github.com/MagicStack/MagicPython)
+- **Pylar:** [miguelgargallo/pylar](https//github.com/miguelgargallo/pylar)
```

Comment:
This is not a Textmate compatible grammar repo. Please see the CONTRIBUTING.md file.